github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -2,18 +2,18 @@
 # This script contains commands to fill out Drupal settings before installation.
 # Executes only for "SQL workflow".
 - name: Getting name for settings (WINDOWS)
-  set_fact: settings_file=/tmp/reinstall_settings.php
-  when: is_windows == true
+  set_fact: settings_file={{ windows_settings_file }}
+  when: is_windows and settings_file is not defined
```

Comment:
We've got troubles with these boolean checks, they are not working as expected
When you'll run ansible-playbook reinstall.yml - they work.
when you arre going to run sh reinstall.sh - they won't work.